hack/release-notes/README.md

```diff
@@ -1,25 +1,30 @@
-# Release notes generator
+Release notes generator
+=======================
 
-This tool generates release notes from all PRs labeled with a specific version. If an issue is linked, it will use that as well.
+This tool generates release notes from all PRs labeled with a specific version. If any issues are linked to a PR, they will be included in the output as well.
 
 PRs labeled with the following are not included:
-- non-issue
-- refactoring
-- docs
-- test
-- ci
-- backport
+- `>non-issue`
+- `>refactoring`
+- `>docs`
+- `>test`
+- `:ci`
+- `backport`
+- `exclude-from-release-notes`
 
-## Usage
 
-```
-go run . $VERSION > outfile
-```
+Prerequisites
+--------------
 
-e.g.
+Create a GitHub token by going to https://github.com/settings/tokens. The token must have `repo:status` and `public_repo` scopes. Enable SSO for the token as well.
+
+
+Usage
+-----
 
 ```
-go run . v1.1.0 > ../../docs/release-notes/1.1.0.asciidoc
-```
+GH_TOKEN=<token> go run main.go <version>
```

Comment:
Not something you introduced but I wonder if it would make sense to use `GITHUB_TOKEN` which we use in other places as well.